### DISCUSSION TITLE
Support commonjs format

### DISCUSSION THREAD

**Discussion by Codpoe** [mdx-js/mdx#1773] [Q&A] [answered]
I’m curious about why mdx@next only provides es module format?

**Comment by ChristianMurphy** [comment, marked as answer]
This is explained some more at https://github.com/unifiedjs/unified/issues/121, mdx is part of a broader ecosystem which has already migrated.

If you're more interested in why ESM read https://blog.sindresorhus.com/get-ready-for-esm-aa53530b3f77
If you're more interested in why not try to dual package read: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#gistcomment-3850849 and https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#gistcomment-3850692
